web3.md

```diff
@@ -531,7 +547,7 @@ WEB3 JSON RPC
                                          +String #rlpEncodeWord( ACCTTO )
                                          +String #rlpEncodeWord( VALUE )
                                          +String #rlpEncodeString( #unparseByteStack( DATA ) )
-                                         +String #rlpEncodeWord( V +Int 27 )
+                                         +String #rlpEncodeWord( V )
```

Comment:
Let's pull out these changes for `+Int 27` into their own PR and test them separately.